python/ray/autoscaler/_private/monitor.py

```diff
@@ -253,6 +253,12 @@ def update_load_metrics(self):
 
         mirror_node_types = {}
         cluster_full = False
+        if (
```

Comment:
This should go away if you rebase right?

Reply:
Because there might be pending actors in gcs server (if enabling gcs actor scheduler), we need to not only check if any worker node has detected `cluster full` (see line 276-281), but also check gcs server's check (this part).